### PR TITLE
Added try/catch and Modified elementScreenshot

### PR DESCRIPTION
**Pull request by vinodkumarsharma276**
1. Added try/catch in PlayWrightExecutor.ts
2. Changed elementScreenshot to take screenshot of full page if element is not visible.
3. Added startTime/endTime/totalTimeTaken in seconds to StoryWright processor.

### CODE REVIEW

File: src/StoryWrightProcessor/PlayWrightExecutor.ts [hunk 1]
```diff
@@ -30,76 +30,136 @@ export class PlayWrightExecutor {
...
+    }
   }
 
   pressKeyAsync = async (selector: string, key: string) => {
-    await this.page.press(selector, key);
+     try{ 
+      await this.page.press(selector, key);
+     }catch (err){
+        console.error("ERROR: pressKey: ", err.message);
+      }
```
Comment (sunilsurana) [inline src/StoryWrightProcessor/PlayWrightExecutor.ts:69]: Alignment doesnt seem right to me. Format the code once from vscode

File: src/StoryWrightProcessor/PlayWrightExecutor.ts [hunk 2]
```diff
@@ -30,76 +30,136 @@ export class PlayWrightExecutor {
...
+      if (selector.charAt(0) === '#') {
+        element = await this.page.$(`id=${selector.substring(1, selector.length)}`)
+      } else {
+        element = await this.page.$(`${selector}`);
+      }
+      const box = await element.boundingBox();
+      await this.page.mouse.move(box.x + box.width / 2, box.y + box.height / 2);
+      await this.page.mouse.down();
+    }catch (err){
+      console.error("ERROR: mouseDown: ", err.message);
```
Comment (sunilsurana) [inline src/StoryWrightProcessor/PlayWrightExecutor.ts:44]: We should also throw the error again from here. And in our main processor log the error about which story failed. That can be helpful in reporting how many stories succeded vs failed.

Comment (vinodkumarsharma276) [reply src/StoryWrightProcessor/PlayWrightExecutor.ts:49]: Added **'throw err'** to all catch blocks

File: src/StoryWrightProcessor/PlayWrightExecutor.ts (hunk 1 above)
Comment (vinodkumarsharma276) [reply src/StoryWrightProcessor/PlayWrightExecutor.ts]: Fixed alignment

**Pull request merged**